.github/workflows/ci.yml

```diff
@@ -35,11 +35,20 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Upgrade sqlite3
+        run: |
+          wget https://www.sqlite.org/2024/sqlite-autoconf-3450300.tar.gz -O /tmp/sqlite.tar.gz
```

Comment:
GHA images do not have the latest version of SQLite yet. We need to build it from source. This may need to be optimised later.

Will be removed once SQLite is added to GHA images

Reply:
For reference https://drupal.slack.com/archives/C62H9CWQM/p1715175337984879